app/controllers/avo/associations_controller.rb

```diff
@@ -132,7 +132,7 @@ def set_reflection_field
     end
 
     def attachment_id
-      params[:related_id] || params.require(:fields).permit(:related_id)[:related_id]
+      params[:related_id] || params.dig(:fields, :related_id)
```

Comment:
`require` and `permit` is not really necessary here, we just want to access the `related_id`